src/test/java/bio/terra/service/iam/sam/SamIamTest.java

```diff
@@ -33,6 +33,9 @@
 @Category(Unit.class)
 public class SamIamTest {
 
+    // TODO: REVIEWERS: does this test do anything useful?
+    //  It seems to be a series of self-fulfilling mocks. I'm tempted to make it disappear.
```

Comment:
I think it would be useful if the TODOs about the mocks not being correct are addressed. It is useful to have a unit test with a mocked out SAM to make sure we are wrapping it correctly, but I agree that this is not doing that in the current state.

Reply:
+1